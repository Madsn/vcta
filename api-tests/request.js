const http = require('http');
const BASE_URL = 'http://localhost:8080'

export default function request(url) {
    return new Promise(resolve => {
        http.get({path: BASE_URL + url, port: 8080}, response => {
            let data = '';
            response.on('data', _data => data += _data);
            response.on('end', () => resolve(data));
        });
    });
}