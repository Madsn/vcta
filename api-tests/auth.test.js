import request from './request';

test('Not possible to get user info when not authenticated', async () => {
  console.log('starting');
  expect.assertions(1);
  const response = await request('/user');
  console.log(response);
  expect(response).toMatchSnapshot();
})