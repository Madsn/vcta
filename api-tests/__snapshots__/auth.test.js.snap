// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not possible to get user info when not authenticated 1`] = `"Oops"`;
